resources/resource_transformers/babel/babel.go

```diff
@@ -56,6 +61,14 @@ func DecodeOptions(m map[string]interface{}) (opts Options, err error) {
 func (opts Options) toArgs() []string {
 	var args []string
 
+	switch opts.SourceMaps {
+	case "true":
```

Comment:
Drop this. Adding these clever defaults is almost always just adding work (documentation etc.). Take either external or inline.

Reply:
Ok, I added a comment. The reason this was there is that babel doesn't understand "external" really.
.sourceMaps must be a boolean, "inline", "both", or undefined
Added "external" and "inline" as the only options. I didn't think "both" was useful.